tool/grind.dart

```diff
@@ -5,8 +5,28 @@ import 'dart:io';
 
 import 'package:grinder/grinder.dart';
 
-main(args) => grind(args);
+import 'package:unscripted/unscripted.dart';
 
+main(args) {
+
+  addTask(new GrinderTask(
+      'bump',
+      taskFunction: () {
+        var releaseType = context.invocation.positionals.first;
```

Comment:
Ah, ok, cool. The 95% case will be the annotations API, which will be nice and terse; makes sense.